datahub-actions/gradle.lockfile

```diff
@@ -1,190 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
Please do not make changes that are unrelated to your PR's core requirements. Making sweeping changes to the repository makes your PR much harder to review and accept.